core/src/executor/evaluate/evaluated.rs

```diff
@@ -89,7 +93,7 @@ impl TryFrom<Evaluated<'_>> for HashMap<String, Value> {
                 Err(EvaluateError::TextLiteralRequired(format!("{v:?}")).into())
             }
             Evaluated::Value(Value::Str(v)) => HashMap::parse_json_object(v.as_str()),
-            Evaluated::Value(Value::Map(v)) => Ok(v),
+            Evaluated::Value(Value::Map(v)) => Ok(HashMap::from_iter(v.into_iter())),
```

Comment:
To convert from IndexMap to HashMap, currently it collects again.
I'm considering which is better between this way and using enum like this.
```rs
// core/src/store/data_row.rs
enum GluMap {
  IndexMap(IndexMap<String, Value>
  HashMap(HashMap<String,Value>
}

pub enum DataRow {
  Vec(Vec<Value>),
  Map(GlueMap)
}
```